ccf_transaction_processor/scripts/ping_test.py

```diff
@@ -74,10 +74,14 @@ def Main() :
         LOG.error('unable to load ccf configuration file {0}'.format(options.ccf_config))
         pass
 
-    member_cert = os.path.join(CCF_Keys, "{}_cert.pem".format(options.member_name))
-    member_key = os.path.join(CCF_Keys, "{}_privk.pem".format(options.member_name))
-    network_cert = config["start"]["network-cert-file"]
-    (host, port) = config["rpc-address"].split(':')
+    network_cert = os.path.join(CCF_Keys, "networkcert.pem")
+
+    if os.environ.get("PDO_LEDGER_URL") is not None:
+        url =  os.environ.get("PDO_LEDGER_URL")
+        _, host_port = url.split('//')
+        (host, port) = host_port.split(':')
+    else :
+        (host, port) = config["rpc-address"].split(':')
 
```

Comment:
can we use the standard URL manipulation libraries to do this? i believe i mentioned that the URL is very brittle with respect to format.

Reply:
Thank you, sure i will replace it with urlparse from urllib.